tests/src/test/java/alluxio/shell/command/LsCommandTest.java

```diff
@@ -103,6 +111,27 @@ public void lsNoAcl() throws IOException, AlluxioException {
   }
 
   /**
+   * Tests ls -h command when security is not enabled.
+   */
+  @Test
+  @LocalAlluxioClusterResource.Config(
+          confParams = {PropertyKey.Name.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "false",
+                  PropertyKey.Name.SECURITY_AUTHENTICATION_TYPE, "NOSASL"})
+  public void lsHumanReadable() throws IOException, AlluxioException {
+    URIStatus[] files = createFiles();
+    mFsShell.run("ls", "-h", "/testRoot");
+    boolean hSize = true;
+    String expected = "";
+    expected += getLsNoAclResultStr("/testRoot/testFileA", files[0].getCreationTimeMs(), hSize, 10,
+            LsCommand.STATE_FILE_IN_MEMORY);
```

Comment:
NIT: indent four spaces for line-break please.
Please refer to google Java code style guide